### PR TITLE
Translations of the indicator.data_not_available message

### PR DESCRIPTION
**Pull request by brockfanning**
This translation is needed in support of this proposal: https://github.com/open-sdg/open-sdg/pull/786

### CODE REVIEW

**Comment by brockfanning** [comment]
@AAmdeen If you are available, your input on this would be greatly appreciated: https://github.com/open-sdg/sdg-translations/pull/137/files#diff-9e50e94f3599b76fcc3bd9128241f9d1R4

The English is: This data is not available. Please choose alternative data to display.

**Comment by brockfanning** [comment]
@holypeony If you are available, your input on this would be greatly appreciated: https://github.com/open-sdg/sdg-translations/pull/137/files#diff-27252f960b1d8c32223e515266853a4bR4

The English version is: This data is not available. Please choose alternative data to display.

**Comment by brockfanning** [comment]
@MoBosse Do you have any corrections to this translations? https://github.com/open-sdg/sdg-translations/pull/137/files#diff-e63aaf796dcb618e3790d190dfa3c3c0R4

**Comment by brockfanning** [comment]
@SDGArmenia do you have any corrections to this translations? https://github.com/open-sdg/sdg-translations/pull/137/files#diff-72089cf958c03cc7789547f2f3dc6bceR4

**Comment by brockfanning** [comment]
@AigulZiff do you have any corrections to these translations?

Kazakh: https://github.com/open-sdg/sdg-translations/pull/137/files#diff-727b643dda9d4b7540983b2951af1f7dR4

Russian: https://github.com/open-sdg/sdg-translations/pull/137/files#diff-c3fe7439c1751747c71e415d01614e31R4

For reference, here is the English: https://github.com/open-sdg/sdg-translations/pull/137/files#diff-e352fc112ad7c7ee39c1b31a23c0f1bfR71

**Comment by MoBosse** [comment]
@brockfanning The [german translation](https://github.com/open-sdg/sdg-translations/pull/137/files#diff-e63aaf796dcb618e3790d190dfa3c3c0R4) is correct. Thank you!

**Comment by brockfanning** [comment]
@jwestw Do you have any corrections to the Chinese translation [here](https://github.com/open-sdg/sdg-translations/pull/137/files#diff-27252f960b1d8c32223e515266853a4bR4)?

For reference, the English source is: "This data is not available. Please choose alternative data to display."

File: translations/zh-Hans/indicator.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 chart: 图表
 clear_all: 全部清除
 clear_selections: 清除选项
+data_not_available: 该数据不可用。 请选择要显示的替代数据。
 dataset_size_warning: 由于图表上显示的数据集过多，请删除一些选项来以便展示。
```
Comment (jwestw) [inline translations/zh-Hans/indicator.yml:5]: This one was a bit strange. I suggest changing it to something like this. 
由于图表上的数据集超过限制所欲不能显示了。请选择少一些。It simply means ‘the datasets on the display have exceeded the limit. Please select fewer’。 

Definitely worth asking a native speaker to look at these

File: translations/zh-Hans/indicator.yml [hunk 2]
```diff
@@ -1,6 +1,7 @@
...
 download_chart: 下载CSV格式的图表
```
Comment (jwestw) [inline translations/zh-Hans/indicator.yml:6]: Maybe this one should be 

'下载图标的CSV数据集' 
literally 'download the chart's CSV dataset'

File: translations/zh-Hans/indicator.yml [hunk 3]
```diff
@@ -1,6 +1,7 @@
 chart: 图表
 clear_all: 全部清除
 clear_selections: 清除选项
+data_not_available: 该数据不可用。 请选择要显示的替代数据。
```
[the remark this answers is not in the document]
Comment (brockfanning) [reply translations/zh-Hans/indicator.yml:4]: @jwestw You would recommend changing it to `数据不存在。 请选择要显示的替代数据。` correct?

[the remark this answers is not in the document]
Comment (holypeony) [reply translations/zh-Hans/indicator.yml:4]: @jwestw @brockfanning It should be 该数据不存在。请选择别的数据来显示。

File: translations/zh-Hans/indicator.yml (hunk 1 above)
Comment (holypeony) [reply translations/zh-Hans/indicator.yml:5]: @jwestw There's no problem with the original translation.

File: translations/zh-Hans/indicator.yml (hunk 2 above)
Comment (holypeony) [reply translations/zh-Hans/indicator.yml:6]: @brockfanning Can you clarify what does "download_chart" mean? I was only given "Download Chart CSV" when I translated this. If it means "Download the chart's CSV dataset," then @jwestw is right.

File: translations/zh-Hans/indicator.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (brockfanning) [reply translations/zh-Hans/indicator.yml]: Thanks very much @holypeony and @jwestw!

File: translations/zh-Hans/indicator.yml (hunk 2 above)
Comment (brockfanning) [reply translations/zh-Hans/indicator.yml:6]: @holypeony @jwestw The English source is "Download Chart CSV". As for the intent, this is the text of a button that appears beneath a bar chart, and when clicked, the user will receive a CSV file containing the statistics that are being visualized in the chart.

That said, I actually was only intending to do the "This data is not available..." translation in this pull-request. If there are corrections on any other translations, I would be happy to make a separate pull-request for them.

**Pull request merged**